pkg/k8shandler/cluster.go

```diff
@@ -208,20 +204,20 @@ func (elasticsearchRequest *ElasticsearchRequest) updateMinMasters() {
 
 	currentMasterCount, err := esClient.GetMinMasterNodes()
 	if err != nil {
-		logrus.Debugf("Unable to get current min master count for cluster %v", cluster.Name)
+		logrus.Debugf("Unable to get current min master count for cluster %q in namespace %q", cluster.Name, cluster.Namespace)
```

Comment:
nit. it is a common pattern for namespaced k8s resources to provide a `String()` method that returns `namespace/name` per default.